Doc/library/sqlite3.rst

```diff
@@ -406,14 +406,22 @@ Connection Objects
 
    .. attribute:: isolation_level
 
-      Get or set the current default isolation level. :const:`None` for autocommit mode or
-      one of "DEFERRED", "IMMEDIATE" or "EXCLUSIVE". See section
-      :ref:`sqlite3-controlling-transactions` for a more detailed explanation.
+      Get or set the current default isolation level.
```

Comment:
"current default" sounds somewhat confusing to me, as it seems a little oxymoronic. is there maybe another way to describe this that doesn't involve putting both those words together?

Reply:
Hah, I didn't notice that one until you pointed it out 😆 I think it survived from the original version.

Reply:
Indeed it did :) —but I figured I'd mention it while you were at it.

Reply:
I just removed "current default". However, we might as well just remove the whole sentence. It does not add much value.

Reply:
I'll remove this first sentence. But I want to do some final adjustments on the `isolation_level` docs before landing this. Bear with me :)